stable_baselines3/common/preprocessing.py

```diff
@@ -117,6 +133,9 @@ def get_obs_shape(observation_space: spaces.Space) -> Tuple[int, ...]:
     elif isinstance(observation_space, spaces.MultiBinary):
         # Number of binary features
         return (int(observation_space.n),)
+    elif isinstance(observation_space, spaces.Dict):
+        return {key: subspace.shape for (key, subspace) in observation_space.spaces.items()}
```

Comment:
I think it should be a recursive call, no?

Reply:
Should this wait for when nested dicts are supported?

Reply:
this is to support not only Box spaces, not for nested Dict (I don't think we should support nested dict)